.github/workflows/api-baseline.yml

```
@@ -0,0 +1,110 @@
+  contents: read
+
+env:
+  FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: "true"
+
+jobs:
+  api-baseline-master:
+    name: API Baseline vs Master (py3.13)
+    runs-on: ubuntu-latest
+    env:
+      POETRY_INSTALL_ARGS: --all-extras --with dev,powermon
+      API_BASELINE_REF: "master"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)